azure-communication-ui/calling/src/androidTest/java/com/azure/android/communication/BaseUiTest.kt

```diff
@@ -33,6 +33,13 @@ internal open class BaseUiTest {
     internal val lobbyErrorHeaderId = R.id.azure_communication_ui_calling_lobby_error_header
     internal val lobbyErrorHeaderText = R.id.azure_communication_ui_lobby_header_error_text
     internal val lobbyErrorHeaderCloseButton = R.id.azure_communication_ui_calling_lobby_error_close_button
+    internal val toastNotificationId = R.id.azure_communication_ui_toast_notification
```

Comment:
Please change to `azure_communication_ui_calling` instead of `azure_communication_ui`. This helps in Contoso to not have conflict if call and hat have same id.